keyboards/ebastler/isometria_75/rev1/chconf.h

```diff
@@ -0,0 +1,714 @@
+/*
```

Comment:
Don't need all of this: 

https://github.com/qmk/qmk_firmware/blob/01c7108db52677c2a4210bcda0afd2b1d3f7619f/platforms/chibios/keyboard-config-templates/chconf.h